packages/wonder-blocks-button/src/components/button-core.tsx

```diff
@@ -362,47 +368,45 @@ export const _generateStyles = (
 
     let newStyles: Record<string, CSSProperties> = {};
     if (kind === "primary") {
-        const boxShadowInnerColor: string = light
-            ? theme.color.bg.primary.inverse
-            : theme.color.bg.primary.default;
+        const focusStyling = {
+            outlineColor: light ? theme.color.bg.primary.default : color,
+            outlineOffset: 2,
+            outlineStyle: "solid",
+            outlineWidth: 2,
```

Comment:
(For my own learning): When do we include values in a theme vs setting the style directly?

Reply:
ohhhh thanks for being curious. This should be part of the theme as it could allow us in the future to configure these (e.g. in Polaris). I'm going to use a theme variable for this.

Reply:
@beaesguerra I've updated the styles to use theme now.